components/map/AGHLeaveIndicator.tsx

```diff
@@ -4,7 +4,7 @@ function AGHLeaveIndicator() {
   return (
     <div className="pointer-events-none absolute left-0 top-0 box-border size-full border-4 border-errorAlert-foreground">
       <div className="absolute bottom-3 left-1/2 -translate-x-1/2 whitespace-nowrap rounded-md bg-errorAlert-foreground p-3">
-        Znajdujesz się poza AGH
+        Znajdujesz się poza obszarem dni otwartych AGH
```

Comment:
Jak jeszcze masz czas to zmniejsz szerokość  baneru, że jesteś poza AGH

![Screenshot_20250410-175350_Chrome.jpg](https://github.com/user-attachments/assets/b848a39f-6dbf-4a86-a676-0ac156c8051b)

Reply:
Tam trzeba wrapa zrobić, żeby tekst się zawijał.

Poza tym, "Dni Otwartych", nie "dni otwartych" (nazwa własna after all)

I teraz pytanie, bo wiadomość nie może zostać tak jak jest, bo to sugeruje że w tym obszarze są całe DO AGH. Zmieniłbym to na "Znajdujesz się poza obszarem Dni Otwartych AGH na WEAIiIB", ale to może być już strasznie długie i trzebaby tekst lekko zmniejszyć